plugins/modules/system/disk_facts.py

```diff
@@ -0,0 +1,181 @@
+#!/usr/bin/python
+#
+# Copyright 2021 VMware, Inc
+#
+# GNU General Public License v3.0+ (see COPYING or https://www.gnu.org/licenses/gpl-3.0.txt)
+
+
+from __future__ import (absolute_import, division, print_function)
+__metaclass__ = type
+
+
+ANSIBLE_METADATA = {'metadata_version': '1.1',
+                    'status': ['preview'],
+                    'supported_by': 'community'}
+
+
+DOCUMENTATION = '''
+---
+module: disk_facts
+version_added: "2.5.0"
+short_description: Get disk facts in the linux OS
```

Comment:
```suggestion
short_description: Get disk facts for Linux/Unix systems
```
Or for which reason is this Linux only, and should not work on macOS, FreeBSD, OpenBSD, ...?

Reply:
it works with macOS not sure about other FreeBSD or OpenBSD. Can you help me how to check that? @felixfontein

Reply:
According to https://pypi.org/project/psutil/ psutil supports:
- Linux
- Windows
- macOS
- FreeBSD, OpenBSD, NetBSD
- Sun Solaris
- AIX

You should probably simply say "Get disk facts", having `psutil` is a requirement is then enough.